fay-base/src/Data/MutMap.hs

```diff
@@ -62,21 +62,21 @@ mutAssocsI :: MutMap a -> Fay [KeyValI a]
 mutAssocsI = ffi "function() { var r = []; for (var k in %1) { r.push({ instance : 'KeyValI', slot1 : k, slot2 : %1[k] }); } return r; }()"
 
 mutClone :: MutMap a -> Fay (MutMap a)
-mutClone = ffi "jQuery['extend']({}, %1)"
+mutClone = ffi "JSON.parse(JSON.stringify(%1))"
```

Comment:
Is it safe to assume that this value can be converted to/from json?

Reply:
I think you can use `Fay$$objConcat` instead of `jQuery.extend`.

Reply:
You are right. It is not safe, when the value is a function or a class.
```Fay$$objConcat``` is better.